src/main/kotlin/no/nav/syfo/client/oppfolgingstilfelle/OppfolgingstilfelleMetric.kt

```diff
@@ -0,0 +1,20 @@
+package no.nav.syfo.client.oppfolgingstilfelle
```

Comment:
Gjør kanskje ikke noe, men nå er dette synlig for alle andre selv om det kun skal brukes i `OppfolgningstilfelleClient`. Må ikke endres, jeg bare lurer på om det finnes en ryddig måte å koble dette sammen der det brukes?😊 Legge det i samme klasse eller samme fil eller noe sånn? 🤔

Reply:
Tenker at det ikke gjør noe at konstantene her er synlige for resten av kodebasen (det er jo val's og ikke var's). Og bra at de er deklarert et sted i nærheten av der de brukes (et alternativ er å samle alle metrics for en app i en kildekodefil, men synes det å dele det opp som her er bedre). Men man kan legge Metric-fila i en egen metrics-pakke. Men her er det jo ikke så mange kildekodefiler i oppfolgingstilfelle-pakken, så da er det mindre viktig synes jeg.